src/components/OfframpSummaryDialog/index.tsx

```diff
@@ -0,0 +1,152 @@
+        {toToken.offrampFeesFixedComponent ? ` + ${toToken.offrampFeesFixedComponent} ${fiatSymbol}` : ''})
+      </p>
+      <p className="flex items-center gap-2">
+        <NetworkIcon network={network} className="w-4 h-4" />
+        <strong>
+          {feesCost} {fiatSymbol}
+        </strong>
+      </p>
+    </div>
+    <div className="flex justify-between mb-2">
+      <p>Quote</p>
+      <p>
+        <strong>
+          <ExchangeRate tokenOutData={tokenOutAmount} fromToken={fromToken} toTokenSymbol={fiatSymbol} />
+        </strong>
+      </p>
+    </div>
+    <div className="flex justify-between">
+      <p>Partner - {anchorUrl ? new URL(anchorUrl).origin : ''}</p>
+    </div>
+  </section>
+);
+
+interface OfframpSummaryDialogProps {
+  fromToken: InputTokenDetails;
+  toToken: OutputTokenDetails;
+  formToAmount: string;
+  fromAmountString: string;
+  visible: boolean;
+  tokenOutAmount: UseTokenOutAmountResult;
+  anchorUrl?: string;
+  onSubmit: () => void;
+  onClose: () => void;
+}
+
+export const OfframpSummaryDialog: FC<OfframpSummaryDialogProps> = ({
+  fromToken,
+  toToken,
+  visible,
+  formToAmount,
+  fromAmountString,
+  tokenOutAmount,
+  anchorUrl,
+  onClose,
+  onSubmit,
+}) => {
+  const { selectedNetwork } = useNetwork();
+  const fromIcon = useGetAssetIcon(fromToken.networkAssetIcon);
+  const toIcon = useGetAssetIcon(toToken.fiat.assetIcon);
+
+  const { feesCost } = useOfframpFees(tokenOutAmount.data?.roundedDownQuotedAmountOut || Big(0), toToken);
+
+  if (!visible) return null;
+  if (!anchorUrl) return null;
+
+  const content = (
+    <div className="flex flex-col justify-center">
+      <AssetDisplay
+        iconAlt={fromToken.networkAssetIcon}
+        symbol={fromToken.assetSymbol}
+        amount={fromAmountString}
+        iconSrc={fromIcon}
+      />
+      <ArrowDownIcon className="w-4 h-4 my-2" />
+      <AssetDisplay amount={formToAmount} symbol={toToken.fiat.symbol} iconSrc={toIcon} iconAlt={toToken.fiat.symbol} />
+      <FeeDetails
+        fiatSymbol={toToken.fiat.symbol}
+        fromToken={fromToken}
+        toToken={toToken}
+        tokenOutAmount={tokenOutAmount}
+        network={selectedNetwork}
+        feesCost={feesCost}
+        anchorUrl={anchorUrl}
+      />
+    </div>
+  );
+  const actions = (
+    // eslint-disable-next-line react/jsx-no-target-blank
+    <a
+      href={anchorUrl}
+      target="_blank"
+      rel="opener" //noopener forbids the use of postMessages.
+      className="btn-vortex-primary btn rounded-xl"
+      style={{ flex: '1 1 calc(50% - 0.75rem/2)' }}
+      onClick={(e) => {
+        e.preventDefault();
+        onSubmit();
+        window.open(anchorUrl, '_blank');
+      }}
+    >
+      Continue with Partner <ArrowTopRightOnSquareIcon className="w-4 h-4" />
+    </a>
+  );
+
+  return (
+    <Dialog content={content} visible={visible} actions={actions} headerText="You're offramping" onClose={onClose} />
```

Comment:
I just realized that we should change this title since we don't want to call it 'offramp' but use the term 'sell' everywhere. 

```suggestion
    <Dialog content={content} visible={visible} actions={actions} headerText="You're selling" onClose={onClose} />
```